python/text-to-speech/README.md

```diff
@@ -0,0 +1,85 @@
+# 🗣️ Text To Speech with Google, Azure and AWS API
+
+A Python cloud function for text to speech synthesis using [Google](https://cloud.google.com/text-to-speech), [Azure](https://azure.microsoft.com/en-us/products/ai-services/text-to-speech) and [AWS](https://docs.aws.amazon.com/polly/latest/dg/API_SynthesizeSpeech.html).
+
+### Supported Providers and Language Codes
+| Providers | Language Code (BCP-47) |
+| ----------- | ----------- |
+| Google      |[Google Language Code](https://cloud.google.com/text-to-speech/docs/voices) |
+| Azure       |[Azure Language Code](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-support?tabs=stt)  |
+| AWS         |[AWS Language Code](https://docs.aws.amazon.com/polly/latest/dg/API_SynthesizeSpeech.html) |
+
+### Example Input:
+```json
+{
+    "provider":"<YOUR_PROVIDER_HERE>",
+    "language":"<YOUR_LANGUAGE_CODE>",
+    "text":"Hello world!"
+}
+```
+### Example output:
+```json
+{
+    "success":true,
+    "audio_bytes":"iVBORw0KGgoAAAANSUhE...o6Ie+UAAAAASU5CYII="
+}
+```
+### Example error output:
+```json
+{
+    "success":false,
+    "error":"Missing API_KEY""
```

Comment:
GitHub highlighting is telling me this is invalid JSON

Reply:
Okay we fixed the extra quote on README.md